background/usercss-install-helper.js

```diff
@@ -96,28 +96,35 @@ bgReady.all.then(() => {
     const u = new URL(url);
     const m = maybeDistro[u.hostname];
     if (!m || m.rx.test(u.pathname)) {
-      openInstallerPage(tabId, url, {});
+      openInstallerPage(tabId, url, {}).catch(console.error);
```

Comment:
Try using a global `unhandledrejection` handler instead.

Reply:
Isn't that a bad idea since it will log all kinds of uncaught promises including 3rd-party libraries?

Reply:
Also I think this event should only be handled at debug time e.g. execute some code in console like
```js
document.addEventListener("unhandledrejection", e => console.dir(e))
```
Instead of coding it in the repo.

Reply:
On the contrary, it sounds like something useful. We don't many external libraries in the background script so if any of them aren't handling rejections then maybe there's something suspicious in their code that needs to be fixed.

Reply:
Another solution is to add the call stack to the error thrown inside `browser` polyfill ([example](https://github.com/violentmonkey/violentmonkey/blob/master/src/common/browser.js#L56)). It won't work in Firefox, obviously, but maybe it prints a usable call stack by default.

In any case, I'm strongly against peppering the code with explicit console calls.